ux.symfony.com/src/Twig/Components/TimelineItem.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Twig\Components;
+
+use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+
+#[AsTwigComponent('TimelineItem')]
+class TimelineItem
+{
+    public array $item;
+
+    public function getContent(): string
+    {
+        return $this->formatContent($this->item['body'] ?? '');
+    }
+
+    private function formatContent(string $content): string
+    {
+        // Replace "### " with "## "
+        $content = preg_replace('/^## /m', '### ', $content);
+
+        // Replace #1234 with a mardown link to the issue
+        $content = preg_replace('/#(\d+)#/', '[#$1](https://github.com/symfony/ux/issues/$1)', $content);
```

Comment:
I think the extra `#` should not be there

```suggestion
        $content = preg_replace('/#(\d+)/', '[#$1](https://github.com/symfony/ux/issues/$1)', $content);
```

Reply:
Thx!